src/transformers/utils/hub.py

```diff
@@ -1081,6 +1093,27 @@ def extract_info_from_url(url):
     return {"repo": cache_repo, "revision": revision, "filename": filename}
 
 
+def create_and_tag_model_card(repo_id, tags=None):
```

Comment:
Let's include the types so someone doesn't have to look into the internals to know what to pass

```suggestion
def create_and_tag_model_card(repo_id: str, tags: Optional[List[str]] = None):
```

Reply:
+1 on static typing :)

Also maybe naming the method `generate_model_card` would be more appropriate as it implies creating + adding metadata to it (and not only tags)